src/remote/manager.go

```diff
@@ -0,0 +1,157 @@
+package remote
+
+import (
+	"fmt"
+	"github.com/iancoleman/orderedmap"
+	"github.com/spf13/cast"
+	"keboola-as-code/src/client"
+	"keboola-as-code/src/local"
+	"keboola-as-code/src/manifest"
+	"keboola-as-code/src/model"
+	"keboola-as-code/src/utils"
+	"sort"
+)
+
+type Manager struct {
+	localManager *local.Manager
+	api          *StorageApi
+}
+
+type Batch struct {
```

Comment:
`Batch` je nejaky objem prace nad API, ktory sa vykona, ked sa zavola `Invoke`, ... obsahuje viacero `Pool`.
Najprv sa vykonaju vsetky operacie nad `branch`, potom `config` a nakoniec sa vykona `Pool` so zmenami nad `Row`.

Je to tak preto, lebo nemozem napr. vytvorit `Row` ak neexistuje `Config`.